test/testspec.js

```diff
@@ -22,21 +22,13 @@ describe('Testing Database', () => {
       expect(results[0].id).to.equal(13);
       expect(results[0].type).to.be.a('string');
     });
-    
-  });
 
-  it("should have 100 records in the interior_features table", function () {
-    connection.query('SELECT * from interior_features', (err, results) => {
-      console.log(results)
-        expect(results).to.equal(100);
-        done();
-    });
   });
 
   it("should have 100 records in the features table", function () {
     connection.query('SELECT * from features', (err, results) => {
       console.log(results)
-        expect(results).to.equal(100);
+        expect(results).to.be.greaterthan(100);
         done();
     });
```

Comment:
I see you just modified this test. However, it's testing your database not your javascript. 
OK if it's your first tests and it bails out of the test suite on failure to prevent the rest of your tests from running

Reply:
Writing more tests now.